### ISSUE TITLE
Filenames with ( and ) generate broken code

### ISSUE THREAD

**Issue by mac-cain13** [mac-cain13/R.swift#97]

**Comment by carlossless** [comment]
:+1: This occurred with having `~` in our file names.

**Comment by mac-cain13** [comment]
Yeah, we should confirm as much as possible to the swift spec to prevent this:

```
Disallowed characters: whitespace, mathematical symbols, arrows, private-use and invalid Unicode points, line- and boxdrawing characters
Special rules: Can't begin with a number
```

**Comment by mac-cain13** [comment]
Will be fixed in the next release, when #107 is merged.
